### ISSUE TITLE
[BUG] Flowise hits query infinite times and does not display the answer in the chat till refreshed

### ISSUE THREAD

**Issue by aadityapattabhiraman** [FlowiseAI/Flowise#3574]
**Describe the bug**
When i hit any query it does not display the reply in chat, 
![Screenshot 2024-11-26 124628](https://github.com/user-attachments/assets/a5846487-f836-4729-ac1a-d7636c7199e0)
when i open terminal i see infinite hits and infinite answers.
![Screenshot 2024-11-26 124558](https://github.com/user-attachments/assets/df0a5045-d34c-4551-9dd5-b703bcb75e00)
When i refresh the site i see the answers
![Screenshot 2024-11-26 124822](https://github.com/user-attachments/assets/367e046f-c487-4cc9-97ab-e90cd5ea4fd5)

**To Reproduce**
give any input to the chat

**Expected behavior**
I expect to see one answer for one query without infinite api hits

**Flow**
[new Chatflow.json](https://github.com/user-attachments/files/17914709/new.Chatflow.json)

**Setup**

-   Installation [`pnpm start`]
-   Flowise Version [`2.1.5`]
-   OS: [Linux]
-   Browser [brave]

**Comment by HenryHengZJ** [comment]
couldnt replicate the issue, are you git cloning the repo, and running using `pnpm start` ?

**Comment by aadityapattabhiraman** [comment]
> couldnt replicate the issue, are you git cloning the repo, and running using `pnpm start` ?
yes

**Comment by aadityapattabhiraman** [comment]
I made an interesting observation.   
* When I tried uninstalling Flowise and starting from scratch, it didn't seem to make a difference, and the bug was still **reproducible**. 
* So, I switched to a new system running **Linux** and followed the exact same steps, and it **worked perfectly**. 

This led me to consider why it didn't work in the first case. It might be because my initial setup was somewhat unconventional. 

* My first environment had **Windows with WSL** installed, and I was running Flowise from within WSL while accessing localhost on Windows. **The bug was reproducible regardless of the installation method used.**

This is definitely something worth thinking about.
